libraries/chain/include/sophiatx/chain/evaluator.hpp

```diff
@@ -4,7 +4,7 @@
 
 namespace sophiatx { namespace chain {
 
-class database;
+class database_interface;
```

Comment:
Evaluators are specific to full node, they don't need to use the abstract interface

Reply:
Generally, to avoid looking into virtual tables for every db call, the abstract interface shall be used only where needed, otherwise the database_object shall be used.